pyproject.toml

```diff
@@ -14,9 +14,9 @@ authors = [
     { name = "Roland Guichard", email = "roland.guichard@pasqal.com" },
     { name = "Joao P. Moutinho", email = "joao.moutinho@pasqal.com"},
 ]
-requires-python = ">=3.8,<3.12"
+requires-python = ">=3.8,<3.13"
 license = {text = "Apache 2.0"}
-version = "1.0.6"
+version = "1.0.7"
```

Comment:
wouldn't say we need to release a new one just for this, but otherwise thanks!

Reply:
sure, reverted.